### ISSUE TITLE
Blobity sticks to top left corner on page route

### ISSUE THREAD

**Issue by rigba** [gmrchk/blobity#19]
**Describe your issue**:
1. install nextjs with typescript
2. install blobity
3. add blobity with useBlobity with any config
4. add a button that routes user to another page
5. move cursor
6. blobity will lock to the top left corner of the screen

If you reload the page after the error fixes itself. Tested on mozilla and edge.

Minimally reproducable repo:
https://github.com/jamesie/blobity-bug

**Blobity config or any additional relevant code used**:

```javascript
  const blobity = useBlobity({
    licenseKey: "opensource",
    magnetic: false,
    color: "#ffffff",
    focusableElementsOffsetX: 4,
    focusableElementsOffsetY: 4,
    zIndex: -1,
    radius: 10,
  });
```

**Before creating this issue, did you think of...**:
- [Y] Have you checked closed issues for similar/related problems?
- [Y] Have you provided all helpful information available?
- [Y] Have you considered creating a demo repository, or a link to your running project where the problem is reproducible?

**Comment by jamesie** [comment]
Also after redirect if you hover on a focusable element the blobity will work as expected and when you hover away the bug goes away. It seems like specifically when the page is routed it triggers the cursor to think it should be at pos 0, 0.

Any ideas for quick fixes while it's being properly fixed would be appreciated

**Comment by jamesie** [comment]
The quick fix for this i discovered is to create a new blobity inside the component that the client is routing to 

```ts
//the page where the bug would occur
import useBlobity from "blobity/lib/react/useBlobity";
import { useRouter } from "next/router";

export default function Bug() {
  const router = useRouter();
  const blobity = useBlobity({
    licenseKey: "opensource",
    magnetic: false,
    color: "#ffffff",
    focusableElementsOffsetX: 4,
    focusableElementsOffsetY: 4,
    zIndex: -1,
    radius: 10,
  });
  return (
    <button
      style={{
        display: "grid",
        alignContent: "center",
        height: "100vh",
      }}
    >
      I am bugged?
    </button>
  );
}

```

**Comment by gmrchk** [comment]
Hi, thank you for taking the time to file the issue and coming up with the workaround too!

This has been a known issue for a while and should be fixed by https://github.com/gmrchk/blobity/pull/13, also available in v0.2.2 on npm. 

I'll close this, but please don't hesitate to reach out in case of further issues!
